cachito/web/models.py

```diff
@@ -273,6 +286,30 @@ def __repr__(self):
             self.id, self.state_name, self.request_id)
 
 
+class EnvironmentVariable(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String, nullable=False)
+    value = db.Column(db.String, nullable=False)
+
+    __table_args__ = (
+        db.UniqueConstraint('name', 'value'),
+    )
+
+    @classmethod
+    def validate_json(cls, name, value):
+        if not isinstance(value, str):
+            raise ValidationError(
+                'The value of environment variables must be a string')
+
+    @classmethod
+    def from_json(cls, name, value):
+        cls.validate_json(name, value)
+        return cls(name=name, value=value)
+
+    def to_json(self):
+        return self.name, self.value
```

Comment:
I see your point from your previous comment. I liked it better without the `to_json` method for this use case.

Reply:
I don't mind it this way. The benefit is that this gives us the opportunity to normalize the key or value in the future. Of course, we don't have a need for that now.

Since a tuple can be serialized as JSON (Array), the name of the method is accurate. 

If you feel strongly about this, I can change it back.

Reply:
That works for me.